dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletAPI.java

```diff
@@ -409,15 +409,6 @@ void cleanField(final Structure structure, final Date deletionDate, final Field
 	 */
 	public void cleanHostField(Structure structure, User user, boolean respectFrontendRoles) throws DotSecurityException, DotDataException, DotMappingException;
 
-	/**
-	 * Finds the next date that a contentlet must be reviewed
-	 * @param content 
-	 * @param user
-	 * @param respectFrontendRoles
-	 * @return Date
-	 * @throws DotSecurityException
-	 */
-	public Date getNextReview(Contentlet content, User user, boolean respectFrontendRoles) throws DotSecurityException;
```

Comment:
why is this removed

Reply:
because review fields are not longer used. Will asked to remove them.